ViewLayer/DebugDisplay/MPPTPage/MPPTView/MpptView.cpp

```diff
@@ -36,9 +36,8 @@ void MpptView::connectMppt(MpptPresenter& mpptPresenter)
 
 void MpptView::mpptReceived(int i, Mppt mppt)
 {
-    double mpptZeroPower = 0;
-    double mpptOnePower = 0;
-    double mpptTwoPower = 0;
+    double mpptPower = 0;
+    static double mpptTotalPower = 0;
```

Comment:
Oooh now I realize what you were trying to do...

So right now, I think that `mpptTotalPower` would just get reset to zero every time `mpptRecieved` is called. Before, the total power would only be the value of the mpptPower at i (the other two would be 0). So to get it so that it has the correct total power, I think we should make `mpptZeroPower` and the others private variables so they get saved, and have this function update them.